enderio-machines/src/main/java/com/enderio/machines/common/io/energy/IMachineEnergyStorage.java

```diff
@@ -0,0 +1,45 @@
+package com.enderio.machines.common.io.energy;
+
+import net.minecraftforge.energy.IEnergyStorage;
+
+/**
+ * Machine energy storage extensions.
+ */
+public interface IMachineEnergyStorage extends IEnergyStorage {
```

Comment:
This interface isn't strictly necessary but I feel like it aids readability. Up for discussion.

Reply:
No longer just for readability, now useful for the new dataslot implementation.